doozerlib/cli/scan_sources.py

```diff
@@ -302,10 +302,10 @@ def _tagged_mosc_id(kubeconfig, version, arch, private) -> str:
     return stdout if stdout else None
 
 
-def _latest_rhcos_build_id(version, arch, private) -> str:
+def _latest_rhcos_build_id(runtime, version, arch, private) -> str:
     """wrapper to return None if anything goes wrong, which will be taken as no change"""
     try:
-        return rhcos.latest_rhcos_build_id(version, arch, private)
+        return rhcos.RHCOSBuildFinder(runtime, version, arch, private).latest_rhcos_build_id()
     except Exception:
```

Comment:
Seems like we could narrow this catch. It hid one bug and could hide another.

Reply:
tough to narrow, there's a lot that can go wrong during the attempt. i guess we could start with obvious and add more as we encounter them.

i also would like to get around to unit testing this specifically...